frontend/public/components/_catalog-item.scss

```diff
@@ -55,3 +55,10 @@ $catalog-item-icon-size-sm: 24px;
     }
   }
 }
+
+.catalog-tile-view-pf-category-body {
+  a:link { text-decoration: none; color: inherit;}
+  a:active { text-decoration: none; color: inherit;}
+  a:visited { text-decoration: none; color: inherit;}
+  a:hover {text-decoration: none; color: inherit;}
+}
```

Comment:
```css
.catalog-tile-view-pf-category-body {
  a, a:active, a:visited, a:hover {
    color: inherit;
    text-decoration: none;
  }
}
```

Reply:
It would probably be better to put our own class that follows BEM on the link instead of using patternfly's